### PR TITLE
Downgrade Vertica Image for Temporary Fix of e2e-leg-9-vcluster

### PR DESCRIPTION
**Pull request by chinhtranvan**
This PR updates the Vertica image to `20240606` to avoid replication errors from the server side. This is a temporary fix, and we will revert this change after fixing the issue on the server side

### CODE REVIEW

**Comment by chinhtranvan** [comment]
e2e-leg-9-vcluster already passed after we downgraded the Vertica image. It is ready for review now, and I hope to merge this PR today to avoid blocking replication progress

**Comment by jizhuoyu** [comment]
> e2e-leg-9-vcluster already passed after we downgraded the Vertica image. It is ready for review now, and I hope to merge this PR today to avoid blocking replication upgrade progress

Could we update the PR title to emphasize that this is a temporary change meant to be reverted later?

**Comment by chinhtranvan** [comment]
> > e2e-leg-9-vcluster already passed after we downgraded the Vertica image. It is ready for review now, and I hope to merge this PR today to avoid blocking replication upgrade progress
> 
> Could we update the PR title to emphasize that this is a temporary change meant to be reverted later?
The PR title already mentioned that this is a temporary fix and will revert this change after the server side fix the issue

**Comment by jizhuoyu** [comment]
> > > e2e-leg-9-vcluster already passed after we downgraded the Vertica image. It is ready for review now, and I hope to merge this PR today to avoid blocking replication upgrade progress
> > 
> > 
> > Could we update the PR title to emphasize that this is a temporary change meant to be reverted later?
> > The PR title already mentioned that this is a temporary fix and will revert this change after the server side fix the issue

Yes I mean update the title, not description, as descriptions are not that obvious to see when you only look at titles of all PRs or search for titles of PRs.

File: .github/workflows/build-images.yml [hunk 1]
```diff
@@ -257,7 +257,7 @@ jobs:
       uses: spilchen/switch-case-action@v2
       id: minimal_vertica_image
       with:
-        default: docker.io/opentext/vertica-k8s-private:latest-minimal-master
+        default: docker.io/opentext/vertica-k8s-private:20240606-minimal
```
Comment (jizhuoyu) [inline .github/workflows/build-images.yml:260]: It's better if we change only the `vertica-image` under `with` clause of `e2e-leg-9-vcluster`, as that's the only test we want to fix? Changing this line in `build-server-minimal` would potentially impact all places where we use server minimal images.

Comment (chinhtranvan) [reply .github/workflows/build-images.yml:260]: I already did that, but we will have trouble with authorization for Docker login to pull the image. We need to sign in to Docker with a username and password to pull the image in build-images.yaml

Comment (jizhuoyu) [reply .github/workflows/build-images.yml]: As we discussed, adding `docker.io` as a prefix of the image name solves the problem.

Comment (chinhtranvan) [reply .github/workflows/build-images.yml]: it seems it works with docker.io :)

**Comment by jizhuoyu** [comment]
looks good, should be able to unblock e2e tests on GitHub CI (assuming the failed udx test is not related to this PR and will be fixed in another PR)

**Review by jizhuoyu — APPROVED** [review]

**Pull request merged**